src/char_table.h

```diff
@@ -32,7 +35,9 @@ struct CharTable
       {
          return(0);
       }
-      if (ch < (int)ARRAY_SIZE(chars))
+      //if (ch < (int)ARRAY_SIZE(chars))
+      // Coverity: CID 76238 (#1 of 1): Out-of-bounds read (OVERRUN)
+      if (ch < CharTable_Length)
```

Comment:
Well, the coverity says about the  values the parameter ch might have.
The test ch < 0 is correct.
The test ch < (int)ARRAY_SIZE(chars) is not enough because ch might have a value (let say 300) which is bigger as the size of the array chars[128]
My proposal is also not enough because it doesn't test the values above 127, which should be
"return(0)"